src/Client/TNTClient.php

```diff
@@ -56,7 +56,8 @@ public function checkZipcodeCityMatch($zipCode, $city)
     public function getDropOffPoints($zipCode, $city = null)
     {
         if (null === $city) {
-            $city = $this->getCitiesGuide($zipCode)->getName();
+            $cities = $this->getCitiesGuide($zipCode);
+            $city = $cities[0]->getName();
```

Comment:
I just created a [PHPBench example](https://github.com/pgrimaud/phpbench/blob/master/benchmarks/FirstElementOfArray.php) about it, for my personal interest.

It seems that current($cities) is 27% slower than $cities[0]. 

If you prefer readability instead of performance (which is not very important in this case), I will update my PR.

Cheers 😄

Reply:
Ahah you're right. But indeed I'd rather go for readability than performance here! Thanks again

Reply:
Done!